includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -315,6 +323,40 @@ private static function buildDescriptorSettings(
 							'default' => $setList[$name] ?? $set['overridedefault']
 						];
 						break;
+					case 'preferences':
+						$preferences = [];
+						foreach( MediaWikiServices::getInstance()->getUserOptionsLookup()->getOptions( $context->getUser() ) as $preference => $val ) {
+							$preferences[$preference] = $preference;
+						}
+						$configs = [
+							'type' => 'multiselect',
+							'options' => isset( $set['options'] ) ? array_merge( $preferences, $set['options'] ) : $preferences,
+							'default' => $setList[$name] ?? $set['overridedefault']
+						];
+						if ( !$disabled ) {
+							$configs['dropdown'] = true;
+						}
+						break;
+					case 'skins':
+  						$enabledSkins = MediaWikiServices::getInstance()->getSkinFactory()->getSkinNames();
+						
+						unset( $enabledSkins['fallback'] );
+						unset( $enabledSkins['apioutput'] );
+						
+						foreach ( $config->get( 'SkipSkins' ) as $skip ) {
+							unset( $enabledSkins[$skip] );
```

Comment:
Would this mean $wgSkipSkins is ManageWiki incompatible?

Reply:
Yes. It doesn't remove the default skins from the drop-down when they are skipped, and if selected resulting in errors.